src/transformState.js

```diff
@@ -4,8 +4,40 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
+const add = 'addProperties';
+const remove = 'removeProperties';
+const clear = 'clear';
```

Comment:
Variable names like this are typically uppercase
```suggestion
const ADD = 'addProperties';
const REMOVE = 'removeProperties';
const CLEAR = 'clear';
```